internal/unit/file.go

```diff
@@ -23,8 +25,8 @@ import (
 	"github.com/coreos/go-systemd/v22/unit"
 )
 
-// New returns a new unit file parsed from raw.
-func New(raw string) (*File, error) {
+// NewUnitFile returns a new unit file parsed from raw.
+func NewUnitFile(raw string) (*File, error) {
```

Comment:
`New` is fine here, `unit.New`, `unit.NewUnitFile` stotters

Reply:
True but maybe `NewFile` then? First, in this package you also find `NewManager` for a new `UnitManager`, and second this func doesn't return a unit but a unit file.

Reply:
`NewFile` sgtm

Reply:
Done!